ftw/linkchecker/command/checking_links.py

```diff
@@ -396,6 +396,14 @@ def main(app, *args):
         setup_plone(app, plone_site_obj)
         email_addresses, base_uri, timeout_config, upload_location = extract_config(config_file)
 
+        # skip linkchecking if the upload location is wrong
+        portal = api.portal.get()
+        try:
+            portal.unrestrictedTraverse(path=upload_location.encode('utf-8'))
+        except Exception as e:
```

Comment:
I think this raises a `KeyError`, please always try to catch the most specific Exception, otherwise try/except blocks may disguise other errors/bugs in the code.

Reply:
Add a changlog entry

Reply:
👍